src/frontend/src/components/common/Header/Header.tsx

```diff
@@ -137,7 +137,7 @@ function Header({ user }: Props) {
       <div data-cy="login" className="other">
         {!isAuthenticated() && (
           <button
-            className="bcgovbtn bcgovbtn__secondary--dark"
+            className="bcgovbtn bcgovbtn__secondary--dark extra-gap"
```

Comment:
Where is `.extra-gap` defined? I don't see that class anywhere, but I do see a `.min-gap` defined below. Did you mean to use that?

Reply:
Ohh no. I intend to remove it only. The default is extra gap now. Thanks for catching!